ur10e_moveit_config/launch/moveit_rviz.launch

```diff
@@ -0,0 +1,15 @@
+<launch>
+
+  <arg name="debug" default="false" />
+  <arg unless="$(arg debug)" name="launch_prefix" value="" />
+  <arg     if="$(arg debug)" name="launch_prefix" value="gdb --ex run --args" />
+
+  <arg name="rviz_config" default="" />
```

Comment:
This is currently missing the previous `config:=true` behavior of loading `$(find ur10e_moveit_config)/launch/moveit.rviz`. I personally would like defaulting this to that config, but this would invert the behavior when providing no argument at all (is there any reason why that was chosen? Just starting an empty rviz doesn't seem too useful).

Reply:
I agree, but let's keep the file's behavior to what it was on the kinetic-devel branch. I'll adapt to get the false/true behavior similar to the kinetic-devel branch.